### PR TITLE
Remove `demo_security_barrier_camera` and update relevant docs

### PR DESCRIPTION
**Pull request by dpigasin**
### Tickets:
 - *57945*

### CODE REVIEW

**Review by ilya-lavrenov — APPROVED** [review]

File: docs/get_started/get_started_linux.md [hunk 1]
```diff
@@ -139,35 +139,6 @@ Throughput: 375.3339402 FPS
...
-./demo_security_barrier_camera.sh -d GPU
-```
-
-When the verification script completes, you see an image that displays the resulting frame with detections rendered as bounding boxes, and text:
-
-![](../img/inference_pipeline_script_lnx.png)
-
-</details>
-
 ### Benchmark Demo Script
```
Comment (vladimir-dudnik) [inline docs/get_started/get_started_linux.md:142]:
```suggestion
### Benchmark Sample Script
```

I would prefer to call IE Samples as samples and OMZ Demos as demos

**Review by vladimir-dudnik — COMMENTED** [review]
can we avoid calling samples as demos?

@andrew-zaytsev please take a look, I think 2022.1 is good point to get naming in order

File: docs/get_started/get_started_linux.md (hunk 1 above)
Comment (ilya-lavrenov) [reply docs/get_started/get_started_linux.md:142]: but the scripts itself are named `demo_***`. Should we rename as well?

Comment (vladimir-dudnik) [reply docs/get_started/get_started_linux.md:142]: @ilya-lavrenov My vote would be for renaming. It seems that names like:
sample_benchmark_app.[bat/sh]
sample_speech_recognition.[bat/sh]
sample_squeezenet_download_convert_run.[bat/sh]
or even
go_sample_benchmark_app.[bat/sh]
go_sample_speech_recognition.[bat/sh]
go_sample_squeezenet.[bat/sh] (go_sample_squeezenet_end2end.[bat/sh] ?)
looks good enough and provide correct positioning of those.

Comment (ilya-lavrenov) [reply docs/get_started/get_started_linux.md:142]: Ok, I agree with renaming. Suggest:
```sh
run_sample_benchmark_app.[bat/sh]
run_sample_speech_recognition.[bat/sh]
run_sample_squeezenet.[bat/sh]
```

Comment (vladimir-dudnik) [reply docs/get_started/get_started_linux.md:142]: @dpigasin could you please rename to suggested: run_sample_... ?

Comment (vladimir-dudnik) [reply docs/get_started/get_started_linux.md:142]: @dpigasin well, I did the required changes

@ilya-lavrenov FYI

**Review by ilya-lavrenov — APPROVED** [review]

File: docs/install_guides/installing-openvino-docker-linux.md [hunk 2]
```diff
@@ -323,34 +323,34 @@ docker run -it --rm --net=host -v /var/tmp:/var/tmp –-ipc=host <image_name>
...
 
-To run the Security Barrier Camera Demo on a specific inference device, run the following commands with the root privileges (additional third-party dependencies will be installed):
+To run the Classification Demo Using SqueezeNet on a specific inference device, run the following commands with the root privileges (additional third-party dependencies will be installed):
 
 **CPU**:
 
 ```sh
 docker run -itu root:root --rm <image_name>
-/bin/bash -c "apt update && apt install sudo && deployment_tools/demo/demo_security_barrier_camera.sh -d CPU -sample-options -no_show"
+/bin/bash -c "apt update && apt install sudo && deployment_tools/demo/demo_squeezenet_download_convert_run.sh -d CPU"
```
Comment (ilya-lavrenov) [inline docs/install_guides/installing-openvino-docker-linux.md:332]: -> sample_squeezenet_run.sh? And in all other places using CTRL + F

Comment (vladimir-dudnik) [reply docs/install_guides/installing-openvino-docker-linux.md:332]: repaced to run_sample_xxx

**Review by vladimir-dudnik — APPROVED** [review]

**Review by vladimir-dudnik — APPROVED** [review]

**Review by ilya-lavrenov — APPROVED** [review]

**Pull request merged**